addons/l10n_dk/models/template_dk.py

```diff
@@ -32,6 +32,7 @@ def _get_dk_res_company(self):
                 'account_journal_early_pay_discount_gain_account_id': 'dk_coa_2720',
                 'account_sale_tax_id': 'tax_s1',
                 'account_purchase_tax_id': 'tax_k1',
+                'check_account_audit_trail': True,
```

Comment:
For reference, this is done in this module mainly for the demo company. 
It won't break if the setting is not present